network/router.go

```diff
@@ -346,3 +358,10 @@ func (r *Router) receiveServerIdentity(c Conn) (*ServerIdentity, error) {
 	log.Lvl4(r.address, "Identity received from", dst.Address)
 	return dst, nil
 }
+
+// AddErrorHandler adds a network error handler function for this router. The functions will be called
+// on network error (e.g. Timeout, Connection Closed, or EOF) with the identity of the faulty
+// remote host as 1st parameter.
+func (r *Router) AddErrorHandler(errorHandler func(*ServerIdentity)) {
```

Comment:
Do we ? Is it a use case to remove it at some point ? We did not do it for `Services`, for `Protocols` we only added it months later, etc...  I'm asking because it brings the trouble of finding the right one to delete. Just pointer comparison ?